book/book.rb

```diff
@@ -0,0 +1,22 @@
+require './item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state, :publish_date, :title
+  attr_reader :id
+
+  def initialize(publisher, cover_state,
+                 publish_date,
+                 archived,
+                 label,
+                 id = Random.rand(1..1000))
+    super(publish_date, archived, id)
```

Comment:
I think this super suppose to contain the same number of arguments as the initialize method from the parent class, because when I was running the project on my side, an error raised claiming the wrong number of arguments, and when I added the remaining arguments everything worked fine. you can check it on your end too.